lib/ts/recipe/thirdparty/components/features/signInAndUp/index.tsx

```diff
@@ -106,22 +107,34 @@ export const SignInAndUpFeature: React.FC<PropType> = (props) => {
             <FeatureWrapper
                 useShadowDom={props.recipe.config.useShadowDom}
                 defaultStore={defaultTranslationsThirdParty}>
-                <Fragment>
-                    {/* No custom theme, use default. */}
-                    {props.children === undefined && (
-                        <SignInAndUpTheme {...childProps} featureState={state} dispatch={dispatch} />
-                    )}
+                <SignInAndUpFeatureWrapper providers={childProps.providers}>
```

Comment:
Instead of this wrapper component, I'd prefer if we used a a custom hook to get the merged provider list.